### PR TITLE
Add withdrawn amount in withdraw success modal

### PR DESCRIPTION
**Pull request by lazarmitic**

### CODE REVIEW

File: packages/tempus-core-services/src/utils/getWithdrawnAmountFromTx.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { ContractTransaction, ethers } from 'ethers';
+import { Decimal } from '../datastructures';
+import ERC20ABI from '../abi/ERC20.json';
+
+export async function getWithdrawnAmountFromTx(
```
Comment (iamvukasin) [inline packages/tempus-core-services/src/utils/getWithdrawnAmountFromTx.ts:5]: The function is very similar to `getDepositAmountFromTx`. Is there a way to have one instead?

Comment (lazarmitic) [reply packages/tempus-core-services/src/utils/getWithdrawnAmountFromTx.ts:5]: Yes, it's possible to create one function to handle both cases, but down the line we need to read receipt data for other actions as well, and I think if we try to create single function for all actions, this function will become unreadable monster.

**Review by iamvukasin — APPROVED** [review]

File: packages/tempus-client_v3/src/components/WithdrawModal/WithdrawModal.tsx [hunk 2]
```diff
@@ -258,7 +268,7 @@ export const WithdrawModal: FC<WithdrawModalProps> = props => {
       )}
       {/* Show success modal if withdraw is finalized */}
       <SuccessModal
-        description={`You have withdrawn [AMOUNT] ${currency.ticker} with ${new Date(
+        description={`You have withdrawn ${withdrawnAmountFormatted} ${currency.ticker} with ${new Date(
```
Comment (ShawTim) [inline packages/tempus-client_v3/src/components/WithdrawModal/WithdrawModal.tsx:271]: we need to extract this as translation like what we did for `DepositModal`

Comment (lazarmitic) [reply packages/tempus-client_v3/src/components/WithdrawModal/WithdrawModal.tsx]: fixed

**Review by ShawTim — DISMISSED** [review]

**Review by ShawTim — APPROVED** [review]

**Pull request merged**